geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -467,10 +582,14 @@ app.DrawController.createStyleFunction_ = function() {
     }
 
     // goog.asserts.assert(goog.isDef(this.get('__style__'));
-    var color = this.get('color');
+    var color = this.get('color') ? this.get('color') : '#FF0000';
```

Comment:
Calling `this.get('color')` twice could be avoided.

```js
var color = this.get('color');
if (!color) {
  color = '#FF000';
}
```

And using

```js
var color = this.get('color') || '#FF0000';
```

would not shock me in that case :)